xlint

```diff
@@ -251,12 +251,12 @@ for template; do
 		done
 	fi
 
-	: ${LICENSE_LIST:=/usr/share/spdx/license.lst}
+	: "${LICENSE_LIST:=/usr/share/spdx/license.lst}"
 	if [ -f "${LICENSE_LIST}" ]; then
 		sed -n 's/license="\(.*\)"/\1/p' "$template" | tr , "\n" | while read -r l; do
-			if echo "$l" | grep -q 'custom:'; then
-				continue
-			fi
+			case "$l" in
+				custom:*|'Public Domain') contine ;;
```

Comment:
typo

Reply:
I did test this. just forgot to stage that change :/